draft-ietf-tls-tls13.md

```diff
@@ -1586,31 +1586,15 @@ will send a "handshake_failure" or "insufficient_security" fatal alert
 TLS also allows several optimized variants of the basic handshake, as
 described below.
 
-
-### Cached Server Configuration
-
-During an initial handshake, the server can provide a ServerConfiguration
-message containing a long-term (EC)DH share. On future
-connections, the client can indicate to the server that it knows the
-server's configuration and if that configuration is valid the server
-can omit both the Certificate or CertificateVerify message (provided
-that a new configuration is not supplied in this handshake).
-
-When a known configuration is used, the server's long-term DHE key is
-combined with the client's ClientKeyShare to produce SS. ES is
-computed as above.  This optimization allows the server to amortize
-the transmission of these messages and the server's signature over
-multiple handshakes, thus reducing the server's computational cost for
-cipher suites where signatures are slower than key agreement,
-principally RSA signatures paired with ECDHE.
-
-
 ### Zero-RTT Exchange
 
-When a cached ServerConfiguration is used, the client can also send
-application data as well as its Certificate and CertificateVerify
-(if client authentication is requested) on its first flight, thus
-reducing handshake latency, as shown below.
+TLS 1.3 supports a "0-RTT" mode in which the client can send
+application data as well as its Certificate and CertificateVerify (if
+client authentication is requested) on its first flight, thus reducing
```

Comment:
Does it make sense to describe client authentication as being requested, or as being permitted in this context?